lib/addons/p5.dom.js

```diff
@@ -525,9 +528,11 @@
 
   /**
    * Creates a dropdown menu &lt;select&gt;&lt;/select&gt; element in the DOM.
+   * It also helps to assign select-box methods to p5.Element when selecting existing select box
    * @method createSelect
-   * @param {boolean} [multiple] true if dropdown should support multiple selections
-   * @return {Object|p5.Element} pointer to p5.Element holding created node
+   * @param {boolean} [multiple] true if dropdown should support multiple selections OR
+   * @param {Object|p5.Element} if wrapping methods to existing select
```

Comment:
This should go in a different block, see "additional signatures" section here: https://github.com/processing/p5.js/wiki/Inline-documentation#additional-signatures